canonical-web-and-design/cookie-policy
# Use vanilla-framework for the pattern styling

## Done
Imported vanilla framework and included the notification pattern instead of recreating locally.

### QA
- Run `npm i` _or yarn_
- Run `gulp build`
- Open the index.html demo file in your browser
- Check the cookie notification is working ok

## Details
Fixes https://github.com/canonical-webteam/cookie-policy/issues/2
